app/controllers/save_pages_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SavePagesController < ApplicationController
```

Comment:
Can you feature-flag this controller?  Something like

```ruby
before_action do
  head :not_found unless ENV["FEATURE_FLAG_SAVE_A_PAGE"] == "enabled"
end
```

Then we'll be able to make this available in staging but not production, to try it out for real.